### ISSUE TITLE
Mac: Running HTML5Setup.sh and it can't find the python file: "No such file or directory" Error

### ISSUE THREAD

**Issue by mollymun** [UnrealEngineHTML5/Documentation#112]
Hi, I am trying to run the HTML5Setup and no matter how many times I redirect, reinstall or try to get it to find the python file, it keeps giving me this error even though you can see the file in that path in the folder:

Engine/Binaries/ThirdParty/Python/Mac/bin/python*: No such file or directory

I'm running on Mac OS, have tried installing python with homebrew and as the app which goes fine but has no effect on fixing this error. Please if someone knows how to fix this, I've been trying to troubleshoot myself for days and can't find anyone else with the same issue. 

Thanks!

**Comment by annachka** [comment]
Hi, I ran into a similar issue on Windows.  It was telling me exactly where it was looking for the python file, and I had a newer version of python installed (it was ignoring my PATH settings).  To fix it, I manually copied the newer python directory that was there and renamed it to what it was looking for and that worked.  I had to do the same for nodejs fyi.  Hope that helps!

**Comment by PMillenros** [comment]
annachka did you manage to package html5 in unreal later? I'm having the same issues, did you copy the emsdk1.39.18/python/3.9.2 and rename the copy to 3.7.4 or something? (windows)

**Comment by annachka** [comment]
Yes, exactly. I was able to get everything to package correctly later (the FPS template project, specifically).  Did you run into another issue? I can check against my setup if that's helpful. Different OS, though.

**Comment by arterror** [comment]
@ mollymum I just removed the * and it worked on Mac
